cogs/moderation.py

```diff
@@ -86,6 +86,11 @@ async def report(self, ctx, user: discord.Member, *reason):
                 
                 print(error)
                 return await ctx.send(embed=discord.Embed(title="Error!", description="Error making the report! Please try again."))
+            
+    @commands.Cog.listener(name='on_message')
+    async def mark_twitter(self, message):
```

Comment:
Are you able to add checks for a twitter link?

Reply:
Explained this on Discord.